build.gradle

```diff
@@ -15,7 +15,6 @@ buildscript {
     dependencies {
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:${kotlin_version}"
         classpath "net.corda.plugins:cordapp:${corda_gradle_plugins_version}"
```

Comment:
If we remove Corda dependencies from module, this one is need to be removed as well

Reply:
I think that it is better to have plugins imported in one place, rather than in every related project.

Reply:
But we removed cordformation & quasar. It's not very clear what we are trying to achieve. These decencies will be included to connector and indy utils too, maybe we can move them only to Corda specific modules then? What do you think?

Reply:
Build script dependencies add some plugins, that are applied at specific places using "apply".
The one that we removed made some negative changes in dependencies of projects.
I want to have all dependencies in one place, if it doesn`t break anything.

Reply:
Let's keep as is for now